lib/elixir/lib/stream.ex

```diff
@@ -423,6 +423,10 @@ defmodule Stream do
   This operation will block the caller by the given interval
   every time a new item is streamed.
 
+  **Do not use this function** to generate number sequences. If
```

Comment:
Can we please remove the bold and say "instead use ..." or "use ... instead". I believe the "use instead" is not idiomatic. :)